thunder/tests/distributed/helper.py

```diff
@@ -181,17 +181,18 @@ def _run(cls, rank, test_name, file_name, pipe, *, fake_pg=False):
             self.rank = rank
             self.file_name = file_name
 
+            local_rank = self.rank % torch.cuda.device_count()
+            torch.cuda.set_device(local_rank)
+            os.environ["LOCAL_RANK"] = str(local_rank)
```

Comment:
maybe unset `LOCAL_RANK` in https://github.com/Lightning-AI/lightning-thunder/blob/b2ce5973d158c71c9853dc06957667971116f61d/thunder/tests/distributed/helper.py#L201-L203?